tests/test_operators.py

```diff
@@ -1235,34 +1235,100 @@ def test_is_complete_date(self):
             .equals(pandas.Series([True, True, True])))
 
     def test_is_unique_set(self):
-        df = pandas.DataFrame.from_dict( {"ARM": ["PLACEBO", "PLACEBO", "A", "A"], "TAE": [1,1,1,2], "LAE": [1,2,1,2], "ARF": [1,2,3,4]})
```

Comment:
We should begin moving all operator tests to their own files. Just to make things more readable